Modules/mathmodule.c

```diff
@@ -2229,11 +2244,11 @@ math_log(PyObject *module, PyObject * const *args, Py_ssize_t nargs)
     if (!_PyArg_CheckPositional("log", nargs, 1, 2))
         return NULL;
 
-    num = loghelper(args[0], m_log);
+    num = loghelper(args[0], m_log, "expected a positive input, got %R");
```

Comment:
Could we use "strictly positive" instead of "positive" input here?

Reply:
What we will gain with a more longer variant?  I don't see difference.